src/DynamoCore/Configuration/DebugSettings.cs

```diff
@@ -2,6 +2,9 @@
 
 namespace Dynamo.Configuration
 {
+    /// <summary>
+    /// DebugSettings is used in Dynamo Logger and UI.
```

Comment:
You have to say why this class is there. Like this class determines the debug menu / context menu....check for this calss functionality and update.